pkg/querier/blocks_store_replicated_set_test.go

```diff
@@ -427,7 +427,7 @@ func TestBlocksStoreReplicationSet_GetClientsFor_ShouldSupportRandomLoadBalancin
 
 	limits := &blocksStoreLimitsMock{storeGatewayTenantShardSize: 0}
 	reg := prometheus.NewPedanticRegistry()
-	s, err := newBlocksStoreReplicationSet(r, randomLoadBalancing, storegateway.NewNopDynamicReplication(), limits, grpcclient.Config{}, log.NewNopLogger(), reg)
+	s, err := newBlocksStoreReplicationSet(r, randomLoadBalancing, storegateway.NewNopDynamicReplication(3), limits, grpcclient.Config{}, log.NewNopLogger(), reg)
```

Comment:
I'm not sure if it matters but it seems like this should be set from the `ringCfg` above.